scss/_modal.scss

```
@@ -175,7 +175,8 @@
 }
 
 @include media-breakpoint-up(lg) {
-  .modal-lg { max-width: $modal-lg; }
+  .modal-lg,
+  .modal-xl { max-width: $modal-lg; }
```

@XhmikosR, I would add some line breaks here:
```suggestion
  .modal-xl { 
    max-width: $modal-lg;
  }
```

Apart from that, this PR looks fine.

---

Agreed. I personally never use one line expressions but here on Bootstrap I didn't enforce it. We might revisit later :)